test/e2e/runTest.js

```diff
@@ -47,11 +47,20 @@ async function recordBrowser(state, test, testPath, browserName) {
   return { success, why };
 }
 
+const pathname = url => {
+  try {
+    new URL(url).pathname;
+  } catch {
+    return "";
+  }
+};
```

Comment:
I bet we can simplify this by simply splitting on the `/` and getting the last part :)